README.rst

```diff
@@ -28,7 +28,7 @@ Luma.LCD
 
 .. image:: https://img.shields.io/maintenance/yes/2020.svg?maxAge=2592000
 
-Python 3 library interfacing LCD displays with the PCD8544, ST7735, ST7567, HT1621, 
+Python 3 library interfacing LCD displays with the PCD8544, ST7735, ST7567, HT1621,
```

Comment:
shouldn't HD44780 be included in the readme as well?

Reply:
Oops.  Pulled text in from intro.rst to re-write that section.